jupiterone/questions/questions.yaml

```
@@ -68,4 +68,23 @@ questions:
         FIND lastpass_user with (active = true AND admin = true AND masterPasswordStrength = 0 AND neverLoggedIn = true) 
   tags:
   - access
-  - iam
\ No newline at end of file
+  - iam
+- id: integration-question-lastpass-weak-password-developers
+  title: Are there developers who contribute code with a weak master password?
+  description:
+    Developers who contributes code (i.e. opens pull requests) are at higher risk 
+    if they have a weak master password (<100) and are actively using LastPass (sitesCount > 0).
+  queries:
+  - name: Developers with Weak Master Password
+    resultsAre: BAD
+    query: |
+        Find unique PR
+        that opened << (bitbucket_user|github_user|gitlab_user)
```

Thoughts on switching this to `User` instead? There shouldn't be any users who are _not_ from a source control system that have an `OPENED` relationship between `PR` and `User`.

```suggestion
        that opened << User
```